### DISCUSSION TITLE
Conversion between lenght and float

### DISCUSSION THREAD

**Discussion by OlshaMB** [slint-ui/slint#2529] [Q&A] [answered]
I need to convert lenght to float or int in slint

**Comment by hunger** [comment, marked as answer]
To quote the [docs](https://slint-ui.com/releases/1.0.0/docs/slint/src/reference/types.html):

> To convert from an integer to a length unit, one can simply multiply by 1px. Or to convert from a length to a float, one can divide by 1phx.

Does that answer your question?
